notebooks/1.0-adm-data-preparation.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 9,
```

Comment:
Jupyter notebooks and Github don't get along well enough. Rerun every file cleanly so that these numbers start at 1 and work their way up.